src/unity/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -106,7 +107,7 @@ float get_accuracy_per_batch(size_t prediction_window, size_t num_classes,
     
     cumulative_per_batch_accuracy += static_cast<float>(num_correct_predictions) / num_predictions;
   }
-  return cumulative_per_batch_accuracy / batch.batch_info.size();
+  return cumulative_per_batch_accuracy;
```

Comment:
Let's rename this helper function, since you've changed the semantics of the return value. 

I honestly don't know what this should really be called? Cumulative chunk accuracy?